src/overlays/actors/ovl_Obj_Tokei_Step/z_obj_tokei_step.c

```diff
@@ -50,9 +51,6 @@ static InitChainEntry sInitChain[] = {
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
 };
 
-extern Gfx D_06000088[];
-extern CollisionHeader D_06000968;
-
 void ObjTokeiStep_SetSysMatrix(ObjTokeiStepPanel* panel) {
     MtxF* sysMatrix;
 
```

Comment:
There's some stuff in the quake function, too: I think everything going in there is probably decimal over hex, and `quake` can be defined where declared.